code/modules/organs/internal/brain.dm

```diff
@@ -137,10 +137,103 @@ GLOBAL_LIST_BOILERPLATE(all_brain_organs, /obj/item/organ/internal/brain)
 	icon_state = "green slime extract"
 	parent_organ = BP_TORSO
 	clone_source = TRUE
+	flags = OPENCONTAINER
 
 /obj/item/organ/internal/brain/slime/is_open_container()
 	return 1
 
+/obj/item/organ/internal/brain/slime/New()
+	..()
+	create_reagents(50)
+
+/obj/item/organ/internal/brain/slime/proc/reviveBody()
+	var/datum/dna2/record/R = new /datum/dna2/record()
+	R.dna = brainmob.dna
+	R.ckey = brainmob.ckey
+	R.id = copytext(md5(brainmob.real_name), 2, 6)
+	R.name = R.dna.real_name
+	R.types = DNA2_BUF_UI|DNA2_BUF_UE|DNA2_BUF_SE
+	R.languages = brainmob.languages
+	R.flavor = list()
+	for(var/datum/modifier/mod in brainmob.modifiers)
+		if(mod.flags & MODIFIER_GENETIC)
+			R.genetic_modifiers.Add(mod.type)
+
+	var/datum/mind/clonemind = brainmob.mind
+
+	if(!istype(clonemind, /datum/mind))	//not a mind
+		return 0
+	if(clonemind.current && clonemind.current.stat != DEAD)	//mind is associated with a non-dead body
+		return 0
+	if(clonemind.active)	//somebody is using that mind
+		if(ckey(clonemind.key) != R.ckey)
+			return 0
+	else
+		for(var/mob/observer/dead/G in player_list)
+			if(G.ckey == R.ckey)
+				if(G.can_reenter_corpse)
+					break
+				else
+					return 0
+
+	for(var/modifier_type in R.genetic_modifiers)	//Can't be revived. Probably won't happen...?
+		if(istype(modifier_type, /datum/modifier/no_clone))
+			return 0
+
+	var/mob/living/carbon/human/H = new /mob/living/carbon/human(get_turf(src), R.dna.species)
+
+	if(!R.dna)
+		H.dna = new /datum/dna()
+		H.dna.real_name = H.real_name
+	else
+		H.dna = R.dna
+
+	H.UpdateAppearance()
+	H.sync_organ_dna()
+	if(!R.dna.real_name)	//to prevent null names
+		R.dna.real_name = "promethean ([rand(0,999)])"
+	H.real_name = R.dna.real_name
+
+	H.nutrition = 260 //Enough to try to regenerate ONCE.
+	H.adjustBruteLoss(40)
+	H.adjustFireLoss(40)
```

Comment:
Has this been tested and confirmed to not double to 80 damage? As most if not all modifications of burn or brute are modified by their burn or brute species mods.

Reply:
I might suggest the nutrition be set to 300, as it's a single regen, and should give enough leeway if someone isn't paying attention as per below.

Reply:
Since they're both doing the same amount of damage, and Prometheans have just as much resistance to brute as they do weakness to burn, the end result is them being put near-crit-but-not-quite regardless.

As for the leeway, they have something like 5-10 seconds to start regenerating, and I wanted regeneration *or* healing, but not both. If you think they should get 300 nutrition, though, I'll gladly make that change!

Reply:
`40/2+40*2 = 100 != 40*2` as a note

Reply:
Oh.

Well, in testing, they were still at least conscious enough to use the regenerate ability, so :woman_shrugging: